src/locales/en/concerts.yaml

```diff
@@ -394,3 +394,16 @@ concerts:
       Haarlem and Amsterdam.
 
       Click [here](/tour) for more information.
+
+  najaar-2021-vredeskerk:
+    title: Najaarsconcert 2021
+    description: |
```

Comment:
Some descriptions for the concerts would be nice, otherwise we end up with empty pages.